nixos/modules/services/search/tika.nix

```diff
@@ -0,0 +1,72 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.tika;
+  inherit (lib)
+    mkIf
+    mkEnableOption
+    mkOption
+    mkPackageOption
+    getExe'
+    types
+    ;
+in
+{
+  meta.maintainers = [ lib.maintainers.drupol ];
+
+  options = {
+    services.tika = {
+      enable = mkEnableOption "Apache Tika server";
+      package = mkPackageOption pkgs "tika" { };
+
+      listenAddress = mkOption {
+        type = types.str;
+        default = "127.0.0.1";
+        example = "0.0.0.0";
+        description = ''
+          The Apache Tika bind address.
+        '';
+      };
+
+      port = mkOption {
+        type = types.port;
+        default = 8000;
+        description = ''
+          The Apache Tike port to listen on
+        '';
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to open the firewall for Apache Tika.
+          This adds `services.tika.port` to `networking.firewall.allowedTCPPorts`.
+        '';
+      };
```

Comment:
Is this implemented ? I would expect something like `networking.firewall.allowedTCPPorts = mkIf cfg.openFirewall [ cfg.settings.port ];` in `config = mkIf cfg.enable { … }`

Reply:
Yeah I forgot it, this is now fixed.